recipes/docker_registry.rb

```diff
@@ -8,32 +8,9 @@
   Chef::Log.warn "could not find the docker registry ip!"
 end
 
-#for docker to push and pull from the registry it needs to trust hops_ca.pem
-case node['platform_family']
-when 'rhel'
-  cert_target = "/etc/pki/ca-trust/source/anchors/#{registry_host}.crt"
-  update_command = "update-ca-trust"
-when 'debian'
-  cert_target = "/usr/local/share/ca-certificates/#{registry_host}.crt"
-  update_command = "update-ca-certificates"
-end
-
-# we are root, using kagent's certificate should be ok
-kagent_crypto_dir = x509_helper.get_crypto_dir(node['kagent']['user'])
-hops_ca = "#{kagent_crypto_dir}/#{x509_helper.get_certificate_bundle_name(node['kagent']['user'])}"
-if ::File.exist?("#{cert_target}") === false && "#{registry_host}" != "local"
-  bash 'add_trust_cert' do
-    user "root"
-    code <<-EOH
-         ln -s #{hops_ca} #{cert_target}
-         #{update_command}
-         EOH
-  end
-
-  #restart docker to take in account the new trusted certs
-  service 'docker' do
-    action [:restart]
-  end
+#restart docker to take in account the new trusted certs
```

Comment:
Just add a comment that the trusted certificates are added at kagent::default so we don't get lost